src/main/java/com/minecolonies/entity/ai/citizen/farmer/EntityAIWorkFarmer.java

```diff
@@ -11,7 +12,9 @@
 import com.minecolonies.util.BlockUtils;
 import com.minecolonies.util.InventoryUtils;
 import com.minecolonies.util.Utils;
+import net.minecraft.block.BlockCarrot;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'net.minecraft.block.BlockCarrot'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)